CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changes to the Mapbox Directions SDK for iOS
 
+## master
+
+* `Directions.apiEndpoint` and `Directions.accessToken` are now publicly readable, making it easier to share this information with other classes which might need it.
```

Comment:
From the point of view of the developer, these properties have never existed until now:

```suggestion
* Added the `Directions.apiEndpoint` and `Directions.accessToken` properties that reflect the values passed into the `Directions` class’s initializers. ([#313](https://github.com/mapbox/MapboxDirections.swift/pull/313))
```